src/Raven.Studio/typescript/components/pages/database/tasks/backups/BackupsPage.tsx

```diff
@@ -274,13 +275,36 @@ export function BackupsPage(props: BackupsPageProps) {
         <div className="flex-grow-1 flex-stretch-items">
             <div className="flex-vertical">
                 {isAdminAccessOrAbove(database) && (
-                    <div className="flex-shrink-0 mb-4">
+                    <div className="flex-shrink-0 hstack gap-2 mb-4">
                         <Button
                             onClick={navigateToRestoreDatabase}
                             title="Navigate to creating a new database from a backup"
                         >
                             <Icon icon="restore-backup" /> Restore a database from a backup
                         </Button>
+                        <FlexGrow />
+                        <AboutView>
+                            <Row>
+                                <Col sm={"auto"}>
+                                    <Icon className="fs-1" icon="backup" color="info" margin="m-0" />
+                                </Col>
+                                <Col>
+                                    <p>
+                                        <strong>Backups</strong> save your data at a specific point in time and allow
```

Comment:
how is it realted to moving js console to react?

Reply:
Just wanted to test the component directly in studio not in storybook